engine/apps/slack/models/slack_usergroup.py

```diff
@@ -134,8 +134,8 @@ def update_members(self, slack_ids):
 
     def get_users_from_members_for_organization(self, organization):
         return organization.users.filter(
+            User.build_permissions_query(RBACPermission.Permissions.CHATOPS_WRITE, organization),
```

Comment:
**NOTE**: `User.build_permissions_query` now returns `django.db.models.Q` object(s) instead of a dict